packages/api-reference/src/components/Content/Introduction/Introduction.vue

```diff
@@ -38,10 +38,8 @@ const formattedSpecTitle = computed(() => {
     <Section class="introduction-section">
       <SectionContent :loading="!info.description && !info.title">
         <div class="badges">
-          <Badge v-if="info.version">
-            {{ info.version }}
-          </Badge>
-          <Badge v-if="specVersion"> OAS {{ specVersion }}</Badge>
+          <Badge v-if="info.version">v{{ info.version }}</Badge>
```

Comment:
```suggestion
          <Badge v-if="info.version">v{{ info.version.replace(/^v/, '') }}</Badge>
```
should we do a quick check in case they already have a v? to avoid a `v v 1`

Reply:
Man, that’s smart!

Let us take a different approach: We should check if the first string is a number, otherwise we could end up with `vstaging` and other funny things.

Also, currently it’s possible a non-string value lands there, so we need to make sure it’s a string.

I’ll add this tomorrow!